docs/telemetry.md

```diff
@@ -22,32 +92,149 @@ An example of the data collected is the following:
 {"key" : "hardware_arch","value" : "x86_64 x86_64"}]}]
 ```
 
+## Telemetry Agent
```

Comment:
I don't think it will be so easy to uninstall TA package. Package manager will complain that it is a dependency of DB and will not take action.

Reply:
But smb can try autoremove on deb, it automatically removes dependencies.

Reply:
TA is the dependency of DB. AFAIK if you remove DB with autoremove, it will remove TA, not the other way. But someone smarter in that matter should give the opinion here :)

Reply:
Kamil,

As TA is hard/mandatory dependency you can not delete it without deleting dependent package. I was even a bit mistaken, you do not need autoremove, simple remove does the job:

`vagrant@d-buster-64-install:~$ sudo apt list --installed|grep percona

WARNING: apt does not have a stable CLI interface. Use with caution in scripts.

percona-release/stable,now 1.0-29.generic all [installed]
percona-server-client/unknown,now 8.0.36-28-1.buster amd64 [installed,automatic]
percona-server-common/unknown,now 8.0.36-28-1.buster amd64 [installed,automatic]
percona-server-server/unknown,now 8.0.36-28-1.buster amd64 [installed]
percona-telemetry-agent/stable,now 1.0.0-1.buster amd64 [installed,automatic]



vagrant@d-buster-64-install:~$ sudo apt remove percona-telemetry-agent -y
Reading package lists... Done
Building dependency tree       
Reading state information... Done
The following packages were automatically installed and are no longer required:
  libmecab2 libnuma1 percona-server-client percona-server-common
Use 'sudo apt autoremove' to remove them.
!!!!!!!! The following packages will be REMOVED:
!!!!!!!!  percona-server-server percona-telemetry-agent
0 upgraded, 0 newly installed, 2 to remove and 80 not upgraded.
After this operation, 614 MB disk space will be freed.
(Reading database ... 30182 files and directories currently installed.)
Removing percona-server-server (8.0.36-28-1.buster) ...
update-alternatives: removing manually selected alternative - switching my.cnf to auto mode
update-alternatives: using /etc/mysql/my.cnf.fallback to provide /etc/mysql/my.cnf (my.cnf) in auto mode
Removing percona-telemetry-agent (1.0.0-1.buster) ...
Stopping Percona Telemetry Agent service...
Processing triggers for man-db (2.8.5-2) ...
Processing triggers for libc-bin (2.28-10+deb10u2) ...



vagrant@d-buster-64-install:~$ sudo apt list --installed|grep percona

WARNING: apt does not have a stable CLI interface. Use with caution in scripts.

percona-release/stable,now 1.0-29.generic all [installed]
percona-server-client/unknown,now 8.0.36-28-1.buster amd64 [installed,auto-removable]
percona-server-common/unknown,now 8.0.36-28-1.buster amd64 [installed,auto-removable]